src/components/Modal/Modal.tsx

```diff
@@ -0,0 +1,55 @@
+import classnames from 'classnames';
+import { px } from '../../utils';
+import CloseIcon from '../../assets/close.svg?react';
+import Button from '../Button/Button';
+
+export interface ModalProps extends React.HTMLAttributes<HTMLElement> {
+  /**
+   * Boolean that will determine if the modal is open
+   */
+  open?: boolean;
```

Comment:
I think that any boolean property should follow a naming convention like `isOpen` https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/boolean-prop-naming.md.  Can you turn this eslint rule on or open an Jira ticket to track that?